cogs/dotastats.py

```diff
@@ -252,6 +254,47 @@ def __init__(self, bot):
 		self.chat_wheel_info = dotabase.get_chat_wheel_infos()
 		self.dota_gif_lock = asyncio.Lock()
 
+	async def get_meta_json(self): 
+		url = 'https://api.opendota.com/api/herostats'
+		async with httpx.AsyncClient() as client:
```

Comment:
instead of using httpx here, I'd suggest using `await httpgetter.get(url)` which is the custom thing I'm using that is kindof like a centralized place to do http get requests (so i can do things like logging etc.). It'll default to converting to json as well.